lib/packageModules.js

```diff
@@ -34,11 +34,27 @@ function zip(directory, name) {
 
   const output = fs.createWriteStream(artifactFilePath);
 
-  const files = glob.sync('**', {
+  const exclude = _.get(this.configuration, 'package.exclude', []);
```

Comment:
The configuration is now separated into the `Configuration` object. You should add a getter for `package` and set a viable default (empty array) _there_ and add some unit tests. Then just use `const { exclude, include } =  this.configuration.package` here instead of bypassing the integrity of the class.